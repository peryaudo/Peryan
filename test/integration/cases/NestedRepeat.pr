repeat 2
	repeat 5
		mes "hiji"
	loop
	mes "hiza"
loop
