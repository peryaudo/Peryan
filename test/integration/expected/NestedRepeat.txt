hiji
hiji
hiji
hiji
hiji
hiza
hiji
hiji
hiji
hiji
hiji
hiza
